contracts/KeepersCounter.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.7;
+
+import "@chainlink/contracts/src/v0.8/interfaces/KeeperCompatibleInterface.sol";
+
+/**
+ * @title The Counter contract
+ * @notice  A keeper-compatible contract that increments counter variable at fixed time intervals
+ */
+contract KeepersCounter is KeeperCompatibleInterface {
```

Comment:
Remove this file, we have `AutomationCounter`

Reply:
Oops, good catch. Shouldn't have been committed, fragment from that refactor to rename.